app/src/main/index.js

```diff
@@ -0,0 +1,281 @@
+'use strict'
```

Comment:
We have `src/index.js` and `main/index.js` for me it's not clear what are the exact responsibilities of the two index.jses 

A little comment at file begin could help a lot.